hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/management/JpaDistributionSetManagement.java

```diff
@@ -757,10 +787,27 @@ public DistributionSet getValid(final long id) {
 
     @Override
     @Transactional
-    public void invalidate(final DistributionSet set) {
-        final JpaDistributionSet jpaSet = (JpaDistributionSet) set;
+    public void invalidate(final DistributionSet distributionSet) {
```

Comment:
We should extend the DistributionSet interface by the #invalidate function.

Reply:
maybe in other PR